.github/workflows/zowe-cli.yml

```diff
@@ -9,13 +9,19 @@ on:
   workflow_dispatch:
     inputs:
       binary-type:
+        type: choice
         description: Specify whether to use a `debug` or a `release` version of the binary
         default: debug
-        required: false
+        options:
+          - debug
+          - release
       test-type:
+        type: choice
         description: Specify whether to run tests using the `binary` or regular `nodejs` executable
         default: binary
-        required: false
+        options:
+          - binary
+          - nodejs
```

Comment:
Also, thanks for making these input fields a dropdown (choice).
It should prevent typos 😋